### DISCUSSION TITLE
Provide shasum file with releases

### DISCUSSION THREAD

**Discussion by jbouter** [AdguardTeam/AdGuardHome#3759] [Ideas]
In order to verify a download was correct, it would be very nice to have a shasum file available amongst the list of downloads on the GitHub releases page. This will also allow for easier installation with configuration management tools such as SaltStack, as they sometimes require shasum verification.

**Comment by ainar-g** [comment]
This is now feature request #3804.

**Reply by jbouter** [reply]
Hi @ainar-g 👋🏻 !

Thank you so much! :-D
